### DISCUSSION TITLE
what remote desktop client can we use to remote pi5?

### DISCUSSION THREAD

**Discussion by EricYin1** [Joshua-Riek/ubuntu-rockchip#102] [General]
I tried armbian ubuntu with nomachine, it gave me black screen

**Comment by goobii01** [comment]
I have similar issue with Ubuntu form Joshua when I tried with XRDP. After long wait with windows remote desktop to connect Ubuntu I get the message "Connection problem giving up".

**Reply by rahul-kurup** [reply]
If you use Ubuntu (Joshua's release) use "Remote sharing" that exists in the settings manager. Follow the instructions and it'll work out of the box

**Reply by goobii01** [reply]
Work sharing will work only if you are logged in to Ubuntu. I want to connect to my OP5 even it is logged out.
Thanks

**Comment by ironiris** [comment]
Modern OSes are sensitive about remote desktops unless they support a virtual desktop environment.
For example, even in Windows 11, remote desktop doesn't work unless real monitor connected to the computer, so when set up mining machine remotely, they used to plug in a dummy plug to trick it into thinking a monitor connected.

**Comment by 0asisX** [comment]
the current solution I have is to use the native ubuntu sharing feature with the following set:

	1.	auto-login enabled
	2.	the following keyring trick to make the remote password persistent while maintaining secured passwords: https://gitlab.gnome.org/GNOME/gnome-remote-desktop/-/issues/27#note_1357752
	3.	install https://extensions.gnome.org/extension/4338/allow-locked-remote-desktop/ to allow remote on login screen (this won’t work after reboot but when it locks after first login 
	4.	use a fake HDMI dongle as without a screen connected desktop sharing doesn’t work at all
